api_app/serializers.py

```diff
@@ -38,8 +40,11 @@
     "ObservableAnalysisSerializer",
     "AnalysisResponseSerializer",
     "multi_result_enveloper",
+    "CustomConfigSerializer",
 ]
 
+# User = get_user_model()
```

Comment:
forgotten here?

Reply:
Left for easy future use. Should I remove?

Reply:
nah, not needed